packages/appstore/src/components/CFDs/cfd-accounts-container.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import CFDDemoAccounts from './cfd-demo-accounts';
+import CFDRealAccounts from './cfd-real-accounts';
+import { isLandingCompanyEnabled, CFD_PLATFORMS, isMobile } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { Text, StaticUrl } from '@deriv/components';
+import { TPlatform, TAccountCategory, TMt5StatusServer, TMt5StatusServerType, TRootStore } from 'Types';
+import { useStores } from 'Stores/index';
+import { CompareAccountsModal } from '@deriv/cfd';
+
+type TCFDAccountsProps = {
+    account_type: TAccountCategory;
+};
+
+type TOpenAccountTransferMeta = {
+    category: string;
+    type?: string;
+};
+
+const CFDAccounts = ({ account_type }: TCFDAccountsProps) => {
```

Comment:
nit suggestion: since the file name is `cfd-accounts-container.tsx` (or we can change the file name to `cfd-accounts` )

```suggestion
const CFDAccountsContainer = ({ account_type }: TCFDAccountsProps) => {
```

Reply:
Sure thing man 👍